### DISCUSSION TITLE
New Release

### DISCUSSION THREAD

**Discussion by jspricke** [Arkq/bluez-alsa#697] [General]
Hi,

can you please tag a new release? Would be great to have the new features like `--volume` in Debian. Thanks for developing bluez-alsa!

**Comment by arkq** [comment]
> can you please tag a new release?

Yes, I'm planning to tag a new release soon. However, few days ago, I've discovered a very nasty stability bug. I'll tag a new release after fixing it (which unfortunately is not trivial). Expect new release in the first half of May :)

EDIT:
https://github.com/arkq/bluez-alsa/releases/tag/v4.2.0

**Comment by seamusdemora** [comment]
Forgive me for asking, but are there instructions for upgrading in one of the wikis?

**Reply by arkq** [reply]
There is no need for any special things to do when doing upgrades between minor versions bumps. Simply get new release, compile it and install, and you should be fine :)

**Reply by seamusdemora** [reply]
OK, thanks - I wasn't sure if this was considered a minor version bump as you described it as 'a very nasty stability bug' earlier.

**Reply by arkq** [reply]
Yes, the problem with all previous versions was that when stopping and starting playback very quickly, it could lockup bluealsa system service (service restart was required). I've been trying to pinpoint that issue for some time, but only recently I was able to reproduce it in a reliable way, so I could fix it :)

**Reply by seamusdemora** [reply]
I just now realized this Q has been closed, so I've DELETED my question here & asked it as a new question.
